app/src/main/java/de/westnordost/streetcomplete/data/StreetCompleteOpenHelper.java

```diff
@@ -140,7 +141,8 @@
 					CreateNoteTable.Columns.ELEMENT_TYPE +	" varchar(255), " +
 					CreateNoteTable.Columns.ELEMENT_ID +	" int, " +
 					CreateNoteTable.Columns.TEXT + 		" text			NOT NULL, " +
-					CreateNoteTable.Columns.QUEST_TITLE + " text" +
+					CreateNoteTable.Columns.QUEST_TITLE + " text, " +
+					CreateNoteTable.Columns.IMAGE_PATHS + " text" +
```

Comment:
**Serialization issue:** If you serialize/deserialize the image paths from a byte[] blob (using `serializer`), this needs to be of type `blob`, not `text`.

Reply:
Done